src/extension/decorations/closing_labels_decorations.ts

```diff
@@ -12,15 +13,24 @@ export class ClosingLabelsDecorations implements vs.Disposable {
 	private activeEditor?: vs.TextEditor;
 	private closingLabels?: as.AnalysisClosingLabelsNotification;
 
-	private readonly decorationType = vs.window.createTextEditorDecorationType({
-		after: {
-			color: new vs.ThemeColor("dart.closingLabels"),
-			margin: "2px",
-		},
-		rangeBehavior: vs.DecorationRangeBehavior.ClosedOpen,
-	});
+	private decorationType!: vs.TextEditorDecorationType;
+	private closingLabelsPrefix!: string;
+
+	private getDecorationType() {
```

Comment:
nit: I'd call this "update", "build" or "create" or something, since "get" seems to imply it returns something, but it doesn't.